features/step_definitions/indexes.js

```diff
@@ -109,7 +109,7 @@ Then(/^I'm able to delete the index named "([^"]*)"$/, function (index, callback
     .catch(error => callback(error));
 });
 
-Then(/^I refresh the index( ".*?")?$/, function (index, callback) {
+Then(/^I refresh the index( "(.+)")?$/, function (index, callback) {
```

Comment:
I don't understand how this regex can match with the nested catch blocks. Also, why use a greedy catch?

```suggestion
Then(/^I refresh the index(?: "(.*?)")?$/, function (index, callback) {
```

Reply:
I didn't know but it does, maybe cucumber apply some magic here :upside_down_face: 
I use greedy catch because I want something between the double quote

Reply:
I just tested and yes, it works while it should not (we should get 2 matches, the first one including the space and quotes).
About greedineess, I don't think this is what you are expecting:

```
> /"(.+)"/.exec('"somevar" then "some other var"')[1]
'somevar" then "some other var'
> /"(.*?)"/.exec('"somevar" then "some other var"')[1]
'somevar'
```